File: Actividad_1/EjercicioN12.java
package Actividad_1;
public class EjercicioN12{
    public static void main(String[] args) {
        double salariobruto, salarioneto, retenciones= 0.125, retencionesT, horassemana=48, valorhora=5000, horasmensuales;
        horasmensuales=horassemana*4;
        salariobruto=horasmensuales*valorhora;
        retencionesT=salariobruto*retenciones;
        salarioneto= salariobruto-retencionesT;
        System.out.println("El salario bruto es: "+salariobruto);
        System.out.println("Las retenciones son: "+ retencionesT);
        System.out.println("El salario neto es: "+ salarioneto);
    }
}

File: Actividad_1/EjercicioN12.py
horassemana=48
valorhora=5000
retencion=0.125
horasmensuales=horassemana*4 #Se multiplica por 4 ya que un mes tiene 4 semanas
salariobruto=horasmensuales*valorhora
retencionT=salariobruto*retencion
salarioneto= salariobruto-retencionT
#Se calculan el salario bruto, neto y las retenciones y se imprime
print(f"El salario bruto es: {salariobruto}") 
print("Las retenciones son:", retencionT)
print("El salario neto es:", salarioneto)